lib/ts/controllers/s-tooltip.ts

```diff
@@ -50,6 +51,23 @@ namespace Stacks {
             super.show(dispatcher);
         }
 
+        /**
+         * Sets up a tooltip popover show after a delay.
+         */
+        scheduleShow(dispatcher: Event | Element | null = null) {
+            this.activeTimeout = setTimeout(() => this.show(dispatcher), 300);
```

Comment:
Recommended to clear an existing `activeTimeout` on call. That way, if this is somehow called again before the delay is finished, we won't get multiple `show` calls (not that it really matters in the end, since it is programmed to return early if shown).

Optional: Personal preference. Commit if desired.

```suggestion
            window.clearTimeout(this.activeTimeout);
            this.activeTimeout = window.setTimeout(() => this.show(dispatcher), 300);
```